### ISSUE TITLE
Whammy bar warble

### ISSUE THREAD

**Issue by snakebyte69** [w3c/musicxml#271]
Whammy Bar Warble is a common technique, it is executed by quickly hitting the bar this produces a rapid vibrato. This would be important to add to the "with-bar" element.

**Comment by snakebyte69** [comment]
Will this make it into MusicXML 4.0?

**Comment by dspreadbury** [comment]
No, it's not in the [MusicXML 4.0 milestone](https://github.com/w3c/musicxml/milestone/2).
